frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -92,9 +45,7 @@ export const selectedUserTemplateUpdater = (options: UpdateOptions) => {
     }),
   );
 
-  if (iGetVmSettingAttribute(state, id, VMSettingsField.USER_TEMPLATE, 'initialized')) {
-    prefillVmTemplateUpdater(options);
-  }
+  prefillVmTemplateUpdater(options);
```

Comment:
pleace can you also remove `initialized` from `vm-settings-tab-initial-state`?

Reply:
done